Control coverage/Feature/AutomationAccounts.md

```diff
@@ -0,0 +1,88 @@
+# AutomationAccounts
+
+**Resource Type:** Microsoft.Automation/automationAccounts
+
+<!-- TOC depthfrom:2 depthto:2 -->
+
+- [Azure_AutomationAccounts_DP_Encrypt_Variables](#azure_automationaccounts_dp_encrypt_variables)
+
+<!-- /TOC -->
+<br/>
+
+## Azure_AutomationAccounts_DP_Encrypt_Variables
+
+### Display Name 
+Automation account variables should be encrypted
+
+### Rationale 
+Encryption helps prevent sensitive data breaches during transfer and storage.
+
+### Control Spec 
+
+> **Passed:** 
+> Microsoft Defender for Cloud (MDC) reports the assessment status for the Automation Account as `Healthy`.
+> <br>
+> OR
+> <br>
+> Any one of the following condition is met.
+> * All associated variables are encrypted.
+> * Count of associated variables is zero.
+> 
+> **Failed:** 
+> Microsoft Defender for Cloud (MDC) reports the assessment status for the Automation Account as either `Unhealthy`, or `NotApplicable` with `cause` - `OffByPolicy` or `Exempt`.
+> <br>
+> OR
+> <br>
+> More than one variable is not encrypted.
+>
+> **Verify:**
+> Microsoft Defender for Cloud (MDC) reports the assessment status for the Automation Account as `Not Applicable` with `cause` other than `OffByPolicy` and `Exempt`.
+>
+> **NotApplicable:**
+> Not Applicable.
+>
+> **Note:** If no Microsoft Defender for Cloud (MDC) assessment is found for the Automation Account, response from the ARM API is considered for the evaluation.
+>
+### Recommendation
+
+- **Azure Portal**
+
+  Go to Azure Portal --> your Automation Account --> Shared Resources --> Variables. As variable's encrypted state cannot be modified after creation, you need to delete that variable and create another variable with the same name and value with encryption enabled. 
+
+- **PowerShell**
+
+	```powershell
+
+    Connect-AzAccount
+
+    Set-AzContext -Subscription <SubscriptionId>
+
+    $variable = Get-AzAutomationVariable -AutomationAccountName <AutomationAccountName> -ResourceGroupName <RGName> -Name <VariableName>
+
+    Remove-AzAutomationVariable -AutomationAccountName $variable.AutomationAccountName -ResourceGroupName $variable.ResourceGroupName -Name $variable.Name
+
+    $value = $variable.Value
```

Comment:
Move this line up and add a comment saying saving current value of the variable.

Reply:
Done